tv_apps/smart-home/js/card_picker.js

```diff
@@ -167,39 +169,43 @@
       return key.toUpperCase();
     },
 
-    _refreshCardButtons: function(folderList, cardList, options) {
+    _refreshCardElements: function(folderList, cardList, options) {
       var candidates = {};
 
-      this.appButtons = [];
+      this.appCardElems = [];
       this.gridView.innerHTML = '';
 
       var that = this;
-      function appendToGridView(appButton) {
-        that.gridView.appendChild(appButton);
-        that.appButtons.push(appButton);
+      function appendToGridView(appCardElem) {
+        that.gridView.appendChild(appCardElem);
+        that.appCardElems.push(appCardElem);
       }
-      function createButtonHelper(card, parentType) {
+      function createCardElemHelper(card, parentType) {
         if(card instanceof Folder) {
           return;
         }
-        var appButton = CardUtil.createCardButton(card, true);
-        return appButton;
+        var nodeElem = document.createElement('div');
+        nodeElem.appendChild(CardUtil.createCardFragment(card, true));
+        that.gridView.appendChild(nodeElem);
```

Comment:
Guess we shouldn't `appendChild` here, or those need to be sorted will be appended twice.